draft-ietf-acme-acme.md

```diff
@@ -2354,7 +2346,6 @@ certificate applicant controls the identifier he claims.  Before issuing a
 certificate, however, there are many other checks that a CA might need to
 perform, for example:
 
-* Has the client agreed to a subscriber agreement?
```

Comment:
Why would we delete this?  CAs do need to check this (e.g., to comply with the BRs), even if the code for it isn't in the `issue()` method in their ACME server.